docs/dev/reference/dca/list.md

```diff
@@ -128,13 +142,13 @@ $GLOBALS['TL_DCA']['tl_example']['list']['operations'] = [
 | route           | Symfony Route Name (`string`)     | {{< version-tag "4.7" >}} The button will redirect to the given Symfony route.                                                               |
 
 {{% notice "note" %}}
-Since Contao **5.0** you do not have to define any settings for standard operations anymore. Instead you can give a list
+Since Contao **5.0** you do not have to define any settings for standard operations anymore. Instead, you can give a list
```

Comment:
```suggestion
{{< version-tag "5.3" >}} You do not have to define any settings for standard operations anymore. Instead, you can give a list
```

Reply:
That propsal is wrong, that's an upstream conflict. Resolved.